### ISSUE TITLE
Implicit declaration of function "PRINK"

### ISSUE THREAD

**Issue by maribu** [simonjhall/dma#1]
Hi, Simon!

While compiling I got the following error (translated to English):

/mnt/raspberrypi-dma/src/dma-build/dmaer.c:335:4: Error: Implicit declaration of function »PRINK« [-Werror=implicit-function-declaration]

This is the first time I looked at the source code, but from skimming of it I would guess this is a typo and PRINTK instead of PRINK was meant.

Thanks for your work.

Regards,
Maribu

**Comment by simonjhall** [comment]
Yep - well spotted. This is a typo, I've since fixed it in my build.
Good to know that some people are compiling my code ;-)

**Comment by maribu** [comment]
Nice to hear that I can help :-)

I working on a PKGBUILD for Arch Linux, this a cooking recipe for packages. This way hopefully a lot of people can automaticly compile your code with only one command. I hope this way people will get easier access and will beta test.

Regards,
Maribu

**Comment by simonjhall** [comment]
There are a number of key features about to go into the kernel module,
supporting the 'next' branch of rpi kernel (v 3.6) and the CMA stuff that
allows a floating split memory split between the CPU and GPU. I expect to
commit these changes within a week, so perhaps make it easy to add a new
version of this module!

Cheers,
Simon
